Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -314,6 +314,8 @@ def bump_formula_pr
 
     new_formula_version = formula_version(formula, requested_spec, new_contents)
 
+    check_duplicate_pull_requests(formula, tap_full_name, new_formula_version.to_s)
```

Comment:
How about extracting the `Library/Homebrew/dev-cmd/bump-formula-pr.rb` and `Library/Homebrew/utils/github.rb` changes into a separate PR? They could be merged immediately.

Reply:
Sure!

Reply:
@MikeMcQuaid I just realized this has already been done here => https://github.com/Homebrew/brew/commit/f684a59fa5f8dfe6c1498bdd35d08d1e1ab60e92

Reply:
Nice. Worth rebasing or merging to include those changes into here.